CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ Remember to align the itemized text with the first line of an item within a list
 
 # jaxlib 0.4.19
 
+* Changes
+  * jaxlib will now always prefer pip-installed NVIDIA CUDA libraries
+    (nvidia-... packages) over any other CUDA installation if they are
+    installed. If this causes problems, the fix is to remove those library
+    packages.
```

Comment:
Maybe include the exact package names to remove. I probably wouldn't know how to find which pip packages this means. You could also link to setup.py as the source of truth: https://github.com/google/jax/blob/4017a7baca3cbe019ffec8df8effa2608d57b94d/setup.py#L130-L138